topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: tutorial_hands_on
+
+title: "Combining datasets after pre-processing"
+subtopic: single-cell-CS
+priority: 2
+
+zenodo_link: 'https://zenodo.org/record/7041120'
+
+questions:
+  - I have some AnnData files from different samples that I want to combine into a single file. How can I combine these and label them within the object?
+
+objectives:
+  - Combine data matrices from different samples in the same experiment
+  - Label the metadata for downstream processing
+
+time_estimation: 1H
+
+key_points:
+  - Create a single scanpy-accessible AnnData object from multiple AnnData files, including relevant cell metadata according to the study design
+  - Retreive partially analysed data from a public repository
+
+tags:
+  - single-cell
+  - 10x
+  - paper-replication
+  - español
+
+contributors:
+  - nomadscientist
+  - pinin4fjords
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Reply:
And oops! I clearly need to add a "translator" role!